tests/CMakeLists.txt

```diff
@@ -100,6 +100,11 @@ if (UNIX)
 endif ()
 
 if (LINUX)
+    if ($ENV{GCCVERSION} STRGREATER "9")
```

Comment:
What is this for?

Reply:
For newer version of gcc like gcc9 it throws more warnings then before. unused-result is thrown when the source code doesn't check the return value of a function call, like here https://github.com/snowflakedb/libsnowflakeclient/blob/master/tests/test_unit_put_fast_fail.cpp#L407. I would fix the source code if it's in the driver, but for test case I think simply ignore the warning would be enough.